napari/layers/points/points.py

```diff
@@ -28,9 +38,9 @@ class Points(Layer):
         broadcastable to the same shape as the data.
     edge_width : float
         Width of the symbol edge in pixels.
-    edge_color : str
+    edge_color : str, array-like, vispy.color.Color, vispy.color.ColorArray
```

Comment:
I'm really not sure if we even still want to support users passing vispy.color.Color or vispy.color.ColorArray objects - I worry it just sets us up for a backwards compatibility break if we ever move away from vispy for very little gained at this point. I don't think that many of our users will be using vispy objects independent of napari, and now it's fairly clear how to get full control over colors

Reply:
I don't have a strong opinion on this matter, but I will say that just from my limited view of the color-related parts in napari, we depend on vispy's color module in many other locations. I have no problem grepping for `vispy` and throwing it away wherever I can, but I suspect this might be harder than it seems.

Reply:
for now if we remove it from doc strings on our models, and remove imports from our models that are easy to pull out or that were added in this PR, that's a start. we can then remove other references gradually

Reply:
I can do that.